### ISSUE TITLE
v-link always replace instead of push when use a string

### ISSUE THREAD

**Issue by rottmann** [quasarframework/quasar#611]
Problem is this line, when ctx.route is a string .replace is always true
```
vnode.context.$router[ctx.route.replace ? 'replace' : 'push'](ctx.route);
```

Version 0.13.9

**Comment by rstoenescu** [comment]
Hi, thanks for reporting this! Will fix it tomorrow and release v0.13.10.

**Comment by rstoenescu** [comment]
Anyone care for a PR? Should be really easy (just check type of `ctx.route` before looking at "replace" prop). I'm overloaded with v0.14 release.

**Comment by rstoenescu** [comment]
Published v0.13.10 with the fix.
